app/templates/gentelella/users/events/_table.html

```diff
@@ -1,6 +1,23 @@
 {% macro events_table(events) %}
-    <div class="table-responsive">
-        <table class="table table-striped">
+
+<script src="{{ url_for('static', filename='vendor/datatables.net/js/jquery.dataTables.min.js') }}"></script>
+<script src="{{ url_for('static', filename='vendor/datatables.net-bs/js/dataTables.bootstrap.min.js') }}"></script>
+<script type = "text/javascript" >
+	 $('#dodo').DataTable({
```

Comment:
if I won't write this
<script src="{{ url_for('static', filename='vendor/datatables.net/js/jquery.dataTables.min.js') }}"></script>
<script src="{{ url_for('static', filename='vendor/datatables.net-bs/js/dataTables.bootstrap.min.js') }}"></script>
then (column widths) is are not changing . It is not able to recognize the datatable.